PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/luckperms/LuckPermsData.java

```diff
@@ -45,57 +47,73 @@ public InspectContainer getPlayerData(UUID uuid, InspectContainer inspectContain
+            inspectContainer.addTable("Meta", metaTable);
+        }
 
-        TableContainer groupTable = new TableContainer(getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID)));
-        groups.forEach(groupTable::addRow);
-        inspectContainer.addTable("Groups", groupTable);
+        List<String> groups = user.getPermissions().stream()
+                                  .filter(Node::isGroupNode)
+                                  .map(Node::getGroupName)
+                                  .sorted()
+                                  .collect(Collectors.toList());
 
-        TableContainer trackTable = new TableContainer(
-                getWithIcon("Track", Icon.called("ellipsis-h").of(Family.SOLID)),
-                getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID))
+        inspectContainer.addValue(
+                getWithIcon("Groups", Icon.called("user-friends").of(Family.SOLID)),
+                new TextStringBuilder().appendWithSeparators(groups, ", ").build()
         );
-        for (Track track : api.getTracks()) {
-            // reduce is used to get the last element
-            String currentGroup = api.getGroups().stream().map(this::getGroupDisplayName).filter(groups::contains).reduce((first, second) -> second).orElse("None");
-            trackTable.addRow(track.getName(), currentGroup);
+
+        Set<Track> tracks = api.getTracks();
+        if (tracks.size() > 0) {
+            TableContainer trackTable = new TableContainer(
+                    getWithIcon("Track", Icon.called("ellipsis-h").of(Family.SOLID)),
+                    getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID))
+            );
+            for (Track track : tracks) {
+                // reduce is used to get the last element
+                String currentGroup = api.getGroups().stream()
+                                         .map(this::getGroupDisplayName).filter(groups::contains)
+                                         .reduce((first, second) -> second).orElse("None");
+                trackTable.addRow(track.getName(), currentGroup);
+            }
+            inspectContainer.addTable("Tracks", trackTable);
         }
-        inspectContainer.addTable("Tracks", trackTable);
 
         return inspectContainer;
     }
 
     @Override
     public AnalysisContainer getServerData(Collection<UUID> uuids, AnalysisContainer analysisContainer) {
+        // There will *always* be atleast 1 group
         TableContainer groupTable = new TableContainer(
                 getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID)),
                 getWithIcon("Weight", Icon.called("weight-hanging").of(Family.SOLID)),
                 getWithIcon("Permissions", Icon.called("list").of(Family.SOLID))
         );
-        for (Group group : api.getGroups()) {
+
+        for (Group group : api.getGroups().stream().sorted(Comparator.comparing(Group::getName)).collect(Collectors.toList())) {
```

Comment:
This line could be optimized by replacing `.collect(Collectors.toList())` with a `.forEach(group -> {`